src/qt/guiutil.h

```diff
@@ -233,7 +233,7 @@ namespace GUIUtil
         void mouseReleaseEvent(QMouseEvent *event);
     };
 
-#if defined(Q_OS_MAC) && QT_VERSION >= 0x050000
+#if defined(Q_OS_MAC)
```

Comment:
Note: Still need to test, but we might be able to remove this Qt workaround entirely.